dev-php/promise/metadata.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
```

Comment:
As you are creating this one please replace my name / mail with yours

Reply:
It's needed for composer, that's why the maintainers of composer are listed here.

Reply:
Ah I didn't know that,
to me the metadata maintainer section is only for itself but I may be wrong.

Reply:
So @GuillaumeSeren do you agree being added here or not? Do you use composer anymore and care about it still?

Reply:
Yes I still work on it,
actually I discuss to add more tests on base packages, so you can add me.